queries/tags.scm

```diff
@@ -0,0 +1,21 @@
+(binary_operator
+    lhs: (identifier) @name
+    operator: "<-"
+    rhs: (function_definition)
+) @definition.function
+
+(binary_operator
```

Comment:
Is it possible for us to detect function "aliases"? For example:

```
fn <- function(x) {}
alias <- fn
```

In other words, some way of checking for assignments where the rhs symbol might have already have a function definition? I'm guessing not since the tags are probably not stateful in any way, but figure I would ask.

Reply:
Not that I am aware of, but I think that is mostly okay. I think this is going to be most useful when looking at package sources, where this is probably somewhat rare